lib/Cli.php

```diff
@@ -583,7 +583,7 @@ protected function serializeComponent(Component $vObj) {
 
         };
 
-        $children = $vObj->children();
+        $children = iterator_to_array($vObj->children());
```

Comment:
I guess this was the culprit of not getting it faster... Did you try ArrayIterator::uksort?